packages/charting/src/components/CommonComponents/CartesianChart.base.tsx

```diff
@@ -59,11 +59,17 @@ export class CartesianChartBase extends React.Component<IModifiedCartesianChartP
       _height: this.props.height || 350,
     };
     this.idForGraph = getId('chart_');
+    /**
+     * In RTL mode, Only graph will be rendered left/right. We need to provide left and right margins manually.
+     * So that, in RTL, left margins becomes right margins and viceversa.
+     * As graph needs to be drawn perfecty, these values consider as default values.
+     * Same margins using for all other cartesian charts. Can be accessible through 'getMargins' call back method.
+     */
     this.margins = {
       top: this.props.margins?.top || 20,
-      right: this.props.margins?.right || 20,
       bottom: this.props.margins?.bottom || 35,
-      left: this.props.margins?.left || 40,
+      right: this._isRtl ? this.props.margins?.left || 40 : this.props.margins?.right || 20,
+      left: this._isRtl ? this.props.margins?.right || 20 : this.props.margins?.left || 40,
```

Comment:
Use `??` to allow specifying a `0` margin (also do this for top and bottom):

```suggestion
      right: this._isRtl ? this.props.margins?.left ?? 40 : this.props.margins?.right ?? 20,
      left: this._isRtl ? this.props.margins?.right ?? 20 : this.props.margins?.left ?? 40,
```

Reply:
Changes done